library/ssl_tls13_generic.c

```diff
@@ -435,41 +439,40 @@ int mbedtls_ssl_parse_signature_algorithms_ext( mbedtls_ssl_context *ssl,
 
     /* Store the received and compatible signature algorithms for later use. */
     ssl->handshake->received_signature_schemes_list =
-        mbedtls_calloc( num_supported_hashes + 1, sizeof(uint32_t) );
+        mbedtls_calloc( num_supported_hashes + 1, sizeof( uint32_t ) );
```

Comment:
Instead of a short dyamically sized heap allocated buffer, I suggest that we choose a static compile-time bound on the size here.